doc-content/enterprise-only/bpmn/bpmn-con.adoc

```diff
@@ -36,7 +36,7 @@ A process is defined or determined by its process definition. It exists in a kno
 | Enter a description of the purpose of the process.
 
 | *Imports*
-| Click to open the *Imports* window and add any data type classes required for your process.
+| Click to open the *Imports* window and add any data type classes required for your process, or you can select an already defined data type.
```

Comment:
```suggestion
| Click to open the *Imports* window.
| Add any data type classes your process requires, or you can select an already defined data type.
```

Reply:
The above statement is available in a table, that's why we can't split it.